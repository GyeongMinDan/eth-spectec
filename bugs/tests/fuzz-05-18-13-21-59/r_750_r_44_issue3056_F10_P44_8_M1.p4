enum bit<8> E {
b = (("a") |-| ({#}))
};
control compute()() {
  apply {
  }
}
