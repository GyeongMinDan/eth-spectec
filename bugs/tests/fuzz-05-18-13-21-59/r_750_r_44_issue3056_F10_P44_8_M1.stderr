likely_hits/r_750_r_44_issue3056_F10_P44_8_M1.p4(2): [--Werror=type-error] error: |-|: cannot be applied to expression '"a"' with type 'string'
b = (("a") |-| ({#}))
        ^
In file: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172
Compiler Bug: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172: Null srcType

